tensorflow_datasets/object_detection/deeplesion.py

```diff
@@ -0,0 +1,272 @@
+            'zipfile40':'https://nihcc.box.com/shared/static/nc6rwjixplkc5cx983mng9mwe99j8oa2.zip',
+
+            'zipfile41':'https://nihcc.box.com/shared/static/rhnfkwctdcb6y92gn7u98pept6qjfaud.zip',
+            'zipfile42':'https://nihcc.box.com/shared/static/7315e79xqm72osa4869oqkb2o0wayz6k.zip',
+            'zipfile43':'https://nihcc.box.com/shared/static/4nbwf4j9ejhm2ozv8mz3x9jcji6knhhk.zip',
+            'zipfile44':'https://nihcc.box.com/shared/static/1lhhx2uc7w14bt70de0bzcja199k62vn.zip',
+            'zipfile45':'https://nihcc.box.com/shared/static/guho09wmfnlpmg64npz78m4jg5oxqnbo.zip',
+            'zipfile46':'https://nihcc.box.com/shared/static/epu016ga5dh01s9ynlbioyjbi2dua02x.zip',
+            'zipfile47':'https://nihcc.box.com/shared/static/b4ebv95vpr55jqghf6bthg92vktocdkg.zip',
+            'zipfile48':'https://nihcc.box.com/shared/static/byl9pk2y727wpvk0pju4ls4oomz9du6t.zip',
+            'zipfile49':'https://nihcc.box.com/shared/static/kisfbpualo24dhby243nuyfr8bszkqg1.zip',
+            'zipfile50':'https://nihcc.box.com/shared/static/rs1s5ouk4l3icu1n6vyf63r2uhmnv6wz.zip',
+
+            'zipfile51':'https://nihcc.box.com/shared/static/7tvrneuqt4eq4q1d7lj0fnafn15hu9oj.zip',
+            'zipfile52':'https://nihcc.box.com/shared/static/gjo530t0dgeci3hizcfdvubr2n3mzmtu.zip',
+            'zipfile53':'https://nihcc.box.com/shared/static/7x4pvrdu0lhazj83sdee7nr0zj0s1t0v.zip',
+            'zipfile54':'https://nihcc.box.com/shared/static/z7s2zzdtxe696rlo16cqf5pxahpl8dup.zip',
+            'zipfile55':'https://nihcc.box.com/shared/static/shr998yp51gf2y5jj7jqxz2ht8lcbril.zip',
+            'zipfile56':'https://nihcc.box.com/shared/static/kqg4peb9j53ljhrxe3l3zrj4ac6xogif.zip',
+            'ann_file':'https://raw.githubusercontent.com/anir16293/Deep-Lesion/master/DL_info.csv'
+    }
+    dl_manager._register_checksums = True
+    paths = dl_manager.download_and_extract(resources)
+    ann_path = paths['ann_file']
+    del paths['ann_file']
+    lut = _build_lut(paths)
+    train_split, val_split, test_split = Split(ann_path).split_generator()
+
+    return [
+        tfds.core.SplitGenerator(
+            name=tfds.Split.TRAIN,
+            num_shards=100,
+            gen_kwargs={
+                "lut": lut,
+                "split": train_split,
+            },
+        ),
+        tfds.core.SplitGenerator(
+            name=tfds.Split.VALIDATION,
+            num_shards=10,
+            gen_kwargs={
+                "lut": lut,
+                "split": val_split,
+            },
+        ),
+        tfds.core.SplitGenerator(
+            name=tfds.Split.TEST,
+            num_shards=10,
+            gen_kwargs={
+                "lut": lut,
+                "split": test_split,
+            },
+        ),
+    ]
+
+
+  def _generate_examples(self, lut, split):
+    """Returns examples
+    :type lut: dict, lookup dictionary to lookup whole paths of series_folder
+    :type split: string, path to read ann files
```

Comment:
split seems to be a list of tuples. Please update the comment

Reply:
> split seems to be a list of tuples. Please update the comment

updated